token/cli/tests/command.rs

```diff
@@ -167,7 +167,7 @@ async fn new_validator_for_test() -> (TestValidator, Keypair) {
             upgrade_authority: Pubkey::new_unique(),
         },
         UpgradeableProgramInfo {
-            program_id: spl_associated_token_account::id(),
+            program_id: spl_associated_token_account_client::id(),
```

Comment:
This is mostly a semantics thing, but do you think it looks a little strange to be using "client" here? It's implied that the ID is for the program that this package is a client to, but technically `client::id()` can also mean the client's ID.

What if instead it was `spl_associated_token_account_client::program::id()` or also `spl_associated_token_account_client::PROGRAM_ID`?

Reply:
we can do `program::id()` if you think that's clearer. What does "client's ID" refers to though?

Reply:
I think it's a bit clearer yeah. It doesn't refer to anything, for me and you, it just might be confusing to people, especially newer-to-Solana devs. This is one of the first programs you encounter.